platforms/kotlin/src/main/kotlin/ai/snips/nlu/ontology/ffi/COntology.kt

```diff
@@ -148,7 +188,7 @@ object CPrecision {
     }
 }
 
-class CSlotValue : Structure(), Structure.ByValue {
+class CSlotValue(p: Pointer) : Structure(p), Structure.ByReference {
```

Comment:
lets hope this change byValue => by ref doesn't come an bite us later (for example in hermes-protocol)